gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/language/translator/TranslateVisitor.java

```diff
@@ -207,6 +208,12 @@ public Void visitTraversalDT(final GremlinParser.TraversalDTContext ctx) {
         return null;
     }
 
+    @Override
+    public Void visitTraversalN(final GremlinParser.TraversalNContext ctx) {
```

Comment:
Could you add a few cases to [GremlinTranslatorTest](https://github.com/apache/tinkerpop/blob/ca5544dade2a95be3a42524a4d1863c2224b91a4/gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/language/translator/GremlinTranslatorTest.java#L134) to ensure `N` is translating correctly?